plugins/gcalendar-homepage/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@backstage/plugin-gcalendar-homepage",
```

Comment:
Thoughts on maybe just removing the `-homepage` part? I see this as a great beginning of a larger gcalendar integration that may grow to have more than just a homepage widget eventually.

Might have gone as far as making it `plugin-gsuite` or something but maybe that's too far - what makes me say that, is that maybe they'll all depend on the same google libs and share some visual style etc.